shell/common/gin_converters/frame_converter.cc

```diff
@@ -18,4 +20,51 @@ v8::Local<v8::Value> Converter<content::RenderFrameHost*>::ToV8(
   return electron::api::WebFrameMain::From(isolate, val).ToV8();
 }
 
+// static
+v8::Local<v8::Value>
+Converter<gin_helper::AccessorValue<content::RenderFrameHost*>>::ToV8(
+    v8::Isolate* isolate,
+    gin_helper::AccessorValue<content::RenderFrameHost*> val) {
+  auto* rfh = val.Value;
```

Comment:
```suggestion
  content::RenderFrameHost* rfh = val.Value;
```

I like to avoid `auto` unless the type is really long, or is already written out in full elsewhere in the same expression